tests_integration/test_data/projects/snowpark_with_default_values/requirements.txt

```diff
@@ -0,0 +1,3 @@
+snowflake-snowpark-python
+syrupy
+#snowcli  # for local development
```

Comment:
Only snowflake-snowpark-python should be ok

Reply:
fixed